<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>UserMgrForm</class>
    <widget class="QWidget" name="UserMgrForm">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>770</width>
                <height>461</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <widget class="QTableWidget" name="tableWidTabEditUser">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>110</y>
                    <width>751</width>
                    <height>311</height>
                </rect>
            </property>
            <column>
                <property name="text">
                    <string>ID</string>
                </property>
            </column>
            <column>
                <property name="text">
                    <string>姓名</string>
                </property>
            </column>
            <column>
                <property name="text">
                    <string>性别</string>
                </property>
            </column>
            <column>
                <property name="text">
                    <string>用户类型</string>
                </property>
            </column>
            <column>
                <property name="text">
                    <string>电子邮箱</string>
                </property>
            </column>
            <column>
                <property name="text">
                    <string>手机号码</string>
                </property>
            </column>
            <column>
                <property name="text">
                    <string>通讯地址</string>
                </property>
            </column>
        </widget>
        <widget class="QLabel" name="labelTitleEditUser">
            <property name="geometry">
                <rect>
                    <x>620</x>
                    <y>0</y>
                    <width>141</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>12</pointsize>
                    <bold>true</bold>
                </font>
            </property>
            <property name="text">
                <string>用户管理-信息维护</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
        </widget>
        <widget class="QGroupBox" name="groupBoxTabEditUserFilter">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>30</y>
                    <width>751</width>
                    <height>71</height>
                </rect>
            </property>
            <property name="title">
                <string>筛选条件（ID精确查询优先于其他筛选，不为空时自动忽略其他条件）</string>
            </property>
            <widget class="QPushButton" name="pBtnTabEditUserClear">
                <property name="geometry">
                    <rect>
                        <x>680</x>
                        <y>20</y>
                        <width>61</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>清空</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pBtnTabEditUserQuery">
                <property name="geometry">
                    <rect>
                        <x>680</x>
                        <y>42</y>
                        <width>61</width>
                        <height>20</height>
                    </rect>
                </property>
                <property name="text">
                    <string>查询</string>
                </property>
            </widget>
            <widget class="QSplitter" name="splitter">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>20</y>
                        <width>661</width>
                        <height>40</height>
                    </rect>
                </property>
                <property name="orientation">
                    <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <widget class="QWidget" name="layWidg">
                    <layout class="QVBoxLayout" name="vLayoutUserId">
                        <item>
                            <widget class="QLabel" name="labelUserId">
                                <property name="text">
                                    <string>账号（ID）</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="lineEditTabEditUserUserId"/>
                        </item>
                    </layout>
                </widget>
                <widget class="QWidget" name="layWidg_2">
                    <layout class="QVBoxLayout" name="vLayoutUsername">
                        <item>
                            <widget class="QLabel" name="labelUsername">
                                <property name="text">
                                    <string>姓名</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="lineEditTabEditUserUsername"/>
                        </item>
                    </layout>
                </widget>
                <widget class="QWidget" name="layWidg_3">
                    <layout class="QVBoxLayout" name="vLayoutGender">
                        <item>
                            <widget class="QLabel" name="labelGender">
                                <property name="text">
                                    <string>性别</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QComboBox" name="comboBoxTabEditUserGender">
                                <property name="minimumSize">
                                    <size>
                                        <width>72</width>
                                        <height>0</height>
                                    </size>
                                </property>
                                <item>
                                    <property name="text">
                                        <string>所有</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>男性</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>女性</string>
                                    </property>
                                </item>
                            </widget>
                        </item>
                    </layout>
                </widget>
                <widget class="QWidget" name="layWidg_4">
                    <layout class="QVBoxLayout" name="vLayoutUserType">
                        <item>
                            <widget class="QLabel" name="labelUserType">
                                <property name="text">
                                    <string>用户类型</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QComboBox" name="comboBoxTabEditUserUserType">
                                <property name="minimumSize">
                                    <size>
                                        <width>120</width>
                                        <height>0</height>
                                    </size>
                                </property>
                                <item>
                                    <property name="text">
                                        <string>所有用户</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>OB-普通读者</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>FB-教职员读者</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>LA-图书管理员</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>UA-用户管理员</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>SA-系统管理员</string>
                                    </property>
                                </item>
                                <item>
                                    <property name="text">
                                        <string>DS-冻结状态</string>
                                    </property>
                                </item>
                            </widget>
                        </item>
                    </layout>
                </widget>
                <widget class="QWidget" name="layWidg_5">
                    <layout class="QVBoxLayout" name="vLayoutEmail">
                        <item>
                            <widget class="QLabel" name="labelEmail">
                                <property name="text">
                                    <string>电子邮箱</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="lineEditTabEditUserEmail"/>
                        </item>
                    </layout>
                </widget>
                <widget class="QWidget" name="layWidg_6">
                    <layout class="QVBoxLayout" name="vLayoutTelephone">
                        <item>
                            <widget class="QLabel" name="labelTelephone">
                                <property name="text">
                                    <string>手机号码</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="lineEditTabEditUserTelephone"/>
                        </item>
                    </layout>
                </widget>
            </widget>
        </widget>
        <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
                <rect>
                    <x>220</x>
                    <y>430</y>
                    <width>341</width>
                    <height>24</height>
                </rect>
            </property>
            <layout class="QHBoxLayout" name="hLayoutX">
                <item>
                    <widget class="QLineEdit" name="lineEditSelectUserId">
                        <property name="enabled">
                            <bool>false</bool>
                        </property>
                        <property name="readOnly">
                            <bool>true</bool>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="pBtnTabEditUserDeleteUser">
                        <property name="text">
                            <string>删除记录</string>
                        </property>
                    </widget>
                </item>
                <item>
                    <widget class="QPushButton" name="pBtnTabEditUserUpdateUser">
                        <property name="text">
                            <string>执行更新</string>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
